src/styles/_components/_step-card.scss

```diff
@@ -22,38 +23,38 @@
 
     &__step {
         width: 100%;
-        margin-bottom: 8px;
+        margin-bottom: 50px;
         @include font("IntervalNextReg");
-        font-size: em(12px);
-        color: color('text', 'subtitle');
+        font-size: em(20px);
+        color: color("primary", "primary");
```

Comment:
You don't need to add the second key if it's `primary` as the second key defaults to `primary`.

```sass
color: color("primary");
```